docs/en/flight_controller/accton-godwit_ga1.md

```diff
@@ -0,0 +1,153 @@
+
+::: info
+This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+:::
+
+## Specifications
+
+### Processor
+
+- STM32H753IIK (Arm® Cortex®-M7 480MHz)
+- STM32F103 (Arm® Cortex®-M3, 72MHz)
+
+### Sensors
+
+- Bosch BMI088 (vibration isolated)
+- TDK InvenSense ICM-42688-P x 2 (one vibration isolated)
+- TDK Barometric Pressure and Temperature Sensor CP-20100 x 2 (one vibration isolated)
+- PNI RM3100 Geomagnetic Sensor (vibration isolated)
+
+### Power
+
+- 4.6V to 5.7V
+
+### External ports
+
+- 2 CAN Buses (CAN1 and CAN2)
+- 3 TELEM Ports (TELEM1, TELEM2 and TELEM3)
+- 2 GPS Ports (GPS1 with safety switch, LED, buzzer, and GPS2)
+- 1 PPM IN
+- 1 SBUS OUT
+- 2 USB Ports (1 TYPE-C and 1 JST GH1.25)
+- 1 10/100Base-T Ethernet Port
+- 1 DSM/SBUS RC
+- 1 UART 4
+- 1 AD&IO Port
+- 2 Debug Ports (1 IO Debug and 1 FMU Debug)
+- 1 SPI6 Bus
+- 4 Power Inputs (Power 1, Power 2, Power C1 and Power C2)
+- 16 PWM Servo Outputs (A1-A8 from FMU and M1-M8 from IO)
+- Micro SD Socket (supports SD 4.1 & SDIO 4.0 in two databus modes: 1 bit (default) and 4 bits)
+
+### Size and Dimensions
+
+- 92.2 (L) x 51.2 (W) x 28.3 (H) mm
+- 77.6g (carrier board with IMU)
+
+## Where to Buy
+
+- [Accton-IoT Godwit](https://www.accton-iot.com/godwit/)
+- [sales@accton-iot.com](sales@accton-iot.com)
+
+## Pinout
+
+![G-A1 Pin definition](../../assets/flight_controller/accton-godwit/ga1/pin_definition.png "G-A1 Pin definition")
+
+## UART Mapping
+
+| Serial# | Protocol  | Port   | Notes      |
+| ------- | --------- | ------ | ---------- |
+| SERIAL1 | Telem1    | UART7  | /dev/ttyS6 |
+| SERIAL2 | Telem2    | UART5  | /dev/ttyS4 |
+| SERIAL3 | GPS1      | USART1 | /dev/ttyS0 |
+| SERIAL4 | GPS2      | UART8  | /dev/ttyS7 |
+| SERIAL5 | Telem3    | USART2 | /dev/ttyS1 |
+| SERIAL6 | UART4     | UART4  | /dev/ttyS3 |
+| SERIAL7 | FMU Debug | USART3 |            |
+| SERIAL8 | OTG2      | USB    |            |
+
+## Wiring Diagram
+
+![G-A1 Wiring](../../assets/flight_controller/accton-godwit/ga1/wiring.png "G-A1 Wiring")
+
+## PWM Output
+
+PWM M1-M8 (IO Main PWM), A1-A8(FMU PWM).
+All these 16 support normal PWM output formats.
+FMU PWM A1-A6 can support DShot and B-Directional DShot.
+A1-A8(FMU PWM) are grouped as:
+
+- Group 1: A1, A2, A3, A4
+- Group 2: A5, A6
+- Group 3: A7, A8
+
+The motor and servo system should be connected to these ports according to the order outlined in the fuselage reference for your carrier.
+
+![G-A1 PWM Motor Servo](../../assets/flight_controller/accton-godwit/ga1/motor_servo.png "G-A1 PWM Motor Servo")
+
+## RC Input
+
+For DSM/SBUS receivers, connect them to the DSM/SBUS interface which provides dedicated 3.3V and 5V power pins respectively, and check above "Pinout" for detailed pin definition.
+PPM receivers should be connected to the PPM interface.
+
+![G-A1 Radio](../../assets/flight_controller/accton-godwit/ga1/radio.png "G-A1 Radio")
+
```

Comment:
FYI only, you might want to note that other RC systems can be connected via any spare telemetry port. 

This is true, but not yet properly documented.